foundry/foundry.py

```diff
@@ -672,30 +657,6 @@ def build(self, spec, globus=False, interval=3, file=False):
                 key_list = key_list + k
             return key_list
 
-        print("Building Data Package")
```

Comment:
@blaiszik what is the purpose of the `build()` function? We don't appear to use it anywhere. 

Aadit, I am wary of cutting out this chunk of the function without more context (also, can you provide your reasoning for axing this chunk?)

Reply:
Oh that chunk of context can't be run ever. The if else statment has return clauses, so the function always returns in that segment. The chunk is never run.

Reply:
@WardLT I assume you still want this deleted, yes? @Aadit-Ambadkar only part of it appears to be deleted based on the diff

Reply:
I only deleted the part that was dead. The rest is still "alive." I only deleted part of the function which could never be accessed (there was a return statement above that)

Reply:
@Aadit-Ambadkar I believe we determined with @blaiszik and @WardLT that the entire function is deprecated, please delete the whole thing unless Ben and Logan disagree

Reply:
removed with newest commits